### DISCUSSION TITLE
Question: Open HaxeFlixel Debugger

### DISCUSSION THREAD

**Discussion by gamerbross** [FunkinCrew/Funkin#4045] [Q&A] [answered]
I already have my debug build, but i have no idea how to open the HaxeFlixel's Debugger, it just doesnt work as always did

**Comment by xmb9** [comment, marked as answer]
If you're 100% sure you compiled the game for debug and not release, the key will either be F2, `, or \ .

**Comment by gamerbross** [comment]
oh thanks, idk why they overwrote the open debugger keys, now you just can with f2
